Jetsnack/benchmark/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <queries>
+        <package android:name="com.example.jetsnack" />
+    </queries>
+    <uses-permission
```

Comment:
Comment why.

Reply:
This is actually not needed anymore as it's part of the library.
You can remove it from here.